packages/runtime-core/src/main/kotlin/elide/runtime/core/internals/graalvm/GraalVMEngine.kt

```diff
@@ -125,19 +177,43 @@ import org.graalvm.polyglot.HostAccess as PolyglotHostAccess
     ): GraalVMEngine {
       val languages = configuration.languages.map { it.languageId }.toTypedArray()
       val builder = Engine.newBuilder(*languages).apply {
+        useSystemProperties(false)
         allowExperimentalOptions(true)
 
+        // stub streams
+        `in`(StubbedInputStream)
+        out(StubbedOutputStream)
+        err(StubbedOutputStream)
```

Comment:
i think this needs to respect `System.getProperty("elide.js.vm.enableStreams", "false")`

Reply:
good catch, i missed that when moving it